app/src/androidTestWebview/java/org/mozilla/focus/activity/DownloadTest.java

```diff
@@ -84,50 +86,59 @@ protected void afterActivityFinished() {
     };
 
     @After
-    public void tearDown() throws Exception {
+    public void tearDown()  {
 
         // If notification is still up, this will take it off screen
         TestHelper.pressBackKey();
 
         mActivityTestRule.getActivity().finishAndRemoveTask();
     }
 
-    private UiObject downloadIcon = TestHelper.mDevice.findObject(new UiSelector()
-            .description("download icon")
-            .enabled(true));
     private UiObject downloadTitle = TestHelper.mDevice.findObject(new UiSelector()
-            .resourceId("org.mozilla.focus.debug:id/title_template")
+            .resourceId(TestHelper.getAppName() + ":id/title_template")
             .enabled(true));
     private UiObject downloadFileName = TestHelper.mDevice.findObject(new UiSelector()
-            .resourceId("org.mozilla.focus.debug:id/download_dialog_file_name")
+            .resourceId(TestHelper.getAppName() + ":id/download_dialog_file_name")
             .enabled(true));
     private UiObject downloadWarning = TestHelper.mDevice.findObject(new UiSelector()
-            .resourceId("org.mozilla.focus.debug:id/download_dialog_warning")
+            .resourceId(TestHelper.getAppName() + ":id/download_dialog_warning")
             .enabled(true));
     private UiObject downloadCancelBtn = TestHelper.mDevice.findObject(new UiSelector()
-            .resourceId("org.mozilla.focus.debug:id/download_dialog_cancel")
+            .resourceId(TestHelper.getAppName() + ":id/download_dialog_cancel")
             .enabled(true));
     private UiObject downloadBtn = TestHelper.mDevice.findObject(new UiSelector()
-            .resourceId("org.mozilla.focus.debug:id/download_dialog_download")
+            .resourceId(TestHelper.getAppName() + ":id/download_dialog_download")
             .enabled(true));
     private UiObject completedMsg = TestHelper.mDevice.findObject(new UiSelector()
-            .resourceId("org.mozilla.focus.debug:id/snackbar_text")
+            .resourceId(TestHelper.getAppName() + ":id/snackbar_text")
             .enabled(true));
 
     @Test
-    public void DownloadFileTest() throws InterruptedException, UiObjectNotFoundException, IOException {
+    public void DownloadFileTest() throws UiObjectNotFoundException  {
+        UiObject downloadIcon;
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+            downloadIcon = TestHelper.mDevice.findObject(new UiSelector()
+                    .resourceId("download")
+                    .enabled(true));
+        } else {
```

Comment:
\[Checkstyle\] ERROR: '\}' at column 9 should be on the same line as the next part of a multi\-block statement \(one that directly contains multiple blocks: if/else\-if/else, do/while or try/catch/finally\)\.